client/my-sites/stats/stats-comments/index.jsx

```diff
@@ -37,6 +37,12 @@ class StatsComments extends Component {
 		siteId: PropTypes.number,
 		siteSlug: PropTypes.string,
 		className: PropTypes.string,
+		gateStatsComments: PropTypes.bool,
+		skipQuery: PropTypes.bool,
+	};
+
+	static defaultProps = {
```

Comment:
We can remove this because if a property is not passed, it will be falsy by default.